packages/cozy-mespapiers-lib/src/components/Papers/HarvestBanner.jsx

```diff
@@ -1,73 +1,47 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+
 import { useQuery, isQueryLoading } from 'cozy-client'
 import { LaunchTriggerCard } from 'cozy-harvest-lib'
 import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
 
-import {
-  buildTriggersQueryByConnectorSlug,
-  buildConnectorsQueryById,
-  buildAccountsQueryByLoginAndSlug
-} from '../../helpers/queries'
-
-const HarvestBanner = ({ papers }) => {
-  const {
-    createdByApp: connectorSlug,
-    sourceAccountIdentifier: connectorAccountIdentifier
-  } = papers?.list?.[0]?.cozyMetadata || {}
+import { buildTriggersQueryByConnectorSlug } from '../../helpers/queries'
 
-  const queryAccounts = buildAccountsQueryByLoginAndSlug({
-    login: connectorAccountIdentifier,
-    slug: connectorSlug,
-    enabled: Boolean(connectorSlug)
-  })
-  const { data: accounts, ...accountsQueryLeft } = useQuery(
-    queryAccounts.definition,
-    queryAccounts.options
-  )
-  const isAccountsLoading = isQueryLoading(accountsQueryLeft)
-  const account = accounts?.[0]
+const HarvestBanner = ({ connector, account }) => {
+  const connectorSlug = connector?.slug
 
   const queryTriggers = buildTriggersQueryByConnectorSlug(
     connectorSlug,
-    Boolean(connectorSlug)
+    Boolean(connectorSlug) && Boolean(account)
   )
   const { data: triggers, ...triggersQueryLeft } = useQuery(
     queryTriggers.definition,
     queryTriggers.options
   )
   const isTriggersLoading = isQueryLoading(triggersQueryLeft)
+
   const trigger = triggers?.find(
     trigger => trigger.message.account === account?._id
   )
 
-  const queryKonnector = buildConnectorsQueryById(
-    `io.cozy.konnectors/${connectorSlug}`
-  )
-  const { data: konnectors, ...konnectorsQueryLeft } = useQuery(
-    queryKonnector.definition,
-    queryKonnector.options
-  )
-  const isKonnectorsLoading = isQueryLoading(konnectorsQueryLeft)
-  const konnector = konnectors?.[0]
-
-  if (
-    !konnector ||
-    isAccountsLoading ||
-    isTriggersLoading ||
-    isKonnectorsLoading
-  ) {
+  if (!connector || !account || isTriggersLoading) {
     return null
   }
 
   return (
     <>
       <LaunchTriggerCard
-        flowProps={{ initialTrigger: trigger, konnector }}
+        flowProps={{ initialTrigger: trigger, konnector: connector }}
```

Comment:
please keep the `k`! 🙏 

(for instance https://github.com/cozy/cozy-flagship-app/pull/617)

Reply:
ok je vais faire ça dans une PR à côté pour pas bloquer celle-ci en cas de problème